base_layer/core/src/base_node/proto/request.proto

```diff
@@ -23,7 +23,7 @@ message BaseNodeServiceRequest {
         // Indicates a FetchBlocksWithHashes request.
         HashOutputs fetch_blocks_with_hashes = 8;
         // Indicates a GetNewBlockTemplate request.
-        bool get_new_block_template = 9;
+        uint64 get_new_block_template = 9;
```

Comment:
Note that this is not backwards compatible with the previous requests. If this change is not correct, then we can leave the old request and create an additional one?

Reply:
Long term we need something along these lines:
https://docs.microsoft.com/en-us/aspnet/core/grpc/versioning

Since this is a consensus breaking branch, we can accept this breaking change now